ListTasks.java

```diff
@@ -0,0 +1,60 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
```

Comment:
place all of them in a selectable list with asked parameters but use some initial data

Reply:
0a02ec02